cookbooks/Function-Calling-OpenAI/typescript/README.md

```diff
@@ -0,0 +1,21 @@
+# Setup
```

Comment:
Should we seperate into two subdirs for python and ts? It might be a bit confusing to have both in the same dir

Reply:
Can add the python in a later PR - don't want to deal with touching any python here to make sure I don't break anything